modules/upi-removing-hosts-from-a-upi-cluster.adoc

```diff
@@ -74,6 +74,18 @@ openshift-machine-api   controller1   externally provisioned              true
 openshift-machine-api   worker1       deprovisioning                      true             57m
 ----
 
+. Delete the host by running the following command when the `BareMetalHost` state changes to `available`:
++
+[source,terminal]
+----
+$ oc delete bmh -n openshift-machine-api <bmh_name>
+----
++
+[NOTE]
+====
+You can run this step without running the previous step. It might take some time for the `BareMetalHost` state to change from `deprovisioning` to `available`. 
```

Comment:
"Previous step" might be tricky here. What if a new step is introduced before the current step?

Why not state "without having to cordon and drain the node?